operator/src/test/resources/expected/developer-strimzi.yml

```diff
@@ -0,0 +1,295 @@
+---
+kind: "Kafka"
+metadata:
+  annotations:
+    org.bf2.operator/desired-broker-replicas: "1"
+  labels:
+    app.kubernetes.io/managed-by: "kas-fleetshard-operator"
+    ingressType: "sharded"
+    managedkafka.bf2.org/strimziVersion: "strimzi-cluster-operator.v0.23.0-4"
+    bf2.org/kafkaInstanceProfileType: "developer"
```

Comment:
I did not notice before but should we negotiate `bf2.org` to be `managedkafka.bf2.org` here for consistency?

Reply:
If changing it, please let me know, so that I can update https://github.com/bf2fc6cc711aee1a0c2a/observability-resources-mk/pull/177 to match.

I was going to suggest also changing to `kafka-instance-profile-type`, but I see there isn't consistency in the already existing labels, so probably not worth worrying about.

Reply:
@rareddy so you are looking for managedkafka.bf2.org/instance-profile-type?

I don't have a strong preference, but I would address consistency beyond these new labels with another issue.  Any label changes we'd want to make will likely just need to duplicate the labels for a while.

Reply:
I was only suggesting these ones for now as these are new labels, why go through another sync point across Control Plane, Kafka Integrations and Fleet Services team later. if we want address it, do it now or leave it be IMO. 

@pb82 FYI..